assets/js/components/key-metrics-setup/KeyMetricsSetupApp.js

```diff
@@ -121,6 +123,15 @@ export default function KeyMetricsSetupApp() {
 
 	const isInitialSetupFlow = !! showProgress;
 
+	useMount( () => {
+		trackEvent(
```

Comment:
Please replace the ternary operators with a simple `if/else` to keep the way we track events consistent (like we did [here](https://github.com/google/site-kit-wp/pull/11776#discussion_r2550002171)).